kolena/workflow/io.py

```diff
@@ -108,7 +109,7 @@ def _dataframe_object_serde(df, serde_fn: Callable[[Any], Any]):
     df_post = pd.DataFrame(columns=columns)
     for column in columns:
         if df.dtypes[column] == "object":
-            df_post[column] = df[column].apply(serde_fn)
+            df_post[column] = df[column].apply(serde_fn).replace(np.nan, None)
```

Comment:
Also added this logic to handle missing data, in case not all datapoints, through manual curation, have same structure.

Reply:
Rolled back this change as behavior between pandas 1.3 and 1.5 are different, and that of 1.3 is not desirable and don't have an alternative to avoid it.